app/assets/javascripts/analytics/init.js.erb

```diff
@@ -15,6 +15,7 @@
     universalId: universalId,
     cookieDomain: cookieDomain,
     allowLinker: true,
+    govukTrackerGifUrl: '<%= asset_path "/static/a" %>',
```

Comment:
As this is served from Fastly, it would be good to put that in a comment here, as well as the details about the content and status code.